tensorboard/uploader/uploader.py

```diff
@@ -157,6 +163,9 @@ def create_experiment(self):
         response = grpc_util.call_with_retries(
             self._api.CreateExperiment, request
         )
+        self._tracker = (
```

Comment:
Per comment below, how about
```
self._tracker = upload_tracker.UploadTracker(self._verbosity)
```
(and all the consequences of that refactoring)

Reply:
Done.